docs/user_guide/assessment.rst

```diff
@@ -704,10 +609,84 @@ The names of the generated functions are of the form
 :code:`fairlearn.metrics.precision_score_group_min`.
 
-.. _control_features_metrics:
 
-Control features for grouped metrics
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Intersecting Groups
+-------------------
+
+The :class:`MetricFrame` class supports intersectional fairness in two ways:
+multiple sensitive features, and control features.
+Both of these can be used simultaneously.
+One important point to bear in mind when performing an intersectional analysis
+is that some of the intersections may have very few members (or even be empty).
+This will affect the confidence interval associated with the computed metrics;
+random noise has a greater effect on smaller groups.
+
+Multiple Sensitive Features
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Multiple sensitive features can be specified when the :class:`MetricFrame`
+is constructed.
+The :attr:`MetricFrame.by_group` property then holds the intersections
+of these groups:
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> import numpy as np
+    >>> import pandas as pd
+    >>> g_2 = [ 8,6,8,8,8,8,6,6,6,8,6,6,6,6,8,6,6,6 ]
+    >>> s_f_frame = pd.DataFrame(np.stack([sf_data, g_2], axis=1),
+    ...                          columns=['SF 0', 'SF 1'])
+    >>> metric_2sf = MetricFrame(metrics=recall_score,
+    ...                          y_true=y_true,
+    ...                          y_pred=y_pred,
+    ...                          sensitive_features=s_f_frame)
+    >>> metric_2sf.overall  # Same as before
+    0.5
+    >>> metric_2sf.by_group
+    SF 0  SF 1
+    a     6       0.000000
+          8       1.000000
+    b     6       0.666667
+          8       0.500000
+    c     6       0.500000
+          8       0.000000
+    Name: recall_score, dtype: float64
+
+If a particular intersection of the sensitive features had no members, then
+the metric would be shown as :code:`NaN` for that intersection.
+Multiple metrics can also be computed at the same time:
+
+.. doctest:: assessment_metrics
+    :options:  +NORMALIZE_WHITESPACE
+
+    >>> from sklearn.metrics import precision_score
+    >>> metric_2sf_multi = MetricFrame(
+    ...     metrics={'precision':precision_score,
+    ...              'recall':recall_score,
+    ...              'count': count},
+    ...     y_true=y_true,
+    ...     y_pred=y_pred,
+    ...     sensitive_features=s_f_frame
+    ... )
+    >>> metric_2sf_multi.overall
+    precision     0.6
+    recall        0.5
+    count        18.0
+    dtype: float64
+    >>> metric_2sf_multi.by_group
+               precision    recall  count
+    SF 0 SF 1
+    a    6      0.000000  0.000000    2.0
+         8      0.500000  1.000000    2.0
+    b    6      1.000000  0.666667    3.0
+         8      1.000000  0.500000    3.0
+    c    6      0.666667  0.500000    6.0
+         8      0.000000  0.000000    2.0
+
+
+Control Features
+^^^^^^^^^^^^^^^^
 
 Control features (sometimes called 'conditional' features) enable more detailed
```

Comment:
There's an example of when control features might be correlated with sensitive features, but it might also be helpful to provide a quick example of what a control feature might be in fairness analysis.

Reply:
Good point. I've taken a simpler solution, and moved the second paragraph up to complete this one, and had the remainder of this paragraph become the second paragraph.